amr-wind/boundary_conditions/wall_models/WallFunction.cpp

```diff
@@ -11,37 +11,134 @@
 
 namespace amr_wind {
 
-WallFunction::WallFunction(const CFDSim& sim) : m_sim(sim), m_mesh(m_sim.mesh())
+WallFunction::WallFunction(CFDSim& sim)
```

Comment:
Had to change it because of `VelPlaneAveragingFine`, it's constructor only takes a non const `sim`. I am using it to track the mean `utau`, which is a metric for checking convergence of the simulation.